client/my-sites/checkout/composite-checkout/wpcom/types/backend/shopping-cart-endpoint.ts

```diff
@@ -217,6 +217,14 @@ export function convertResponseCartToRequestCart( {
 	} as RequestCart;
 }
 
+export function getProductSlug( cart: ResponseCart, uuidToRemove: string ): string {
+	const filteredProduct = cart.products.filter( ( product ) => {
+		return product.uuid === uuidToRemove;
+	} );
+
+	return filteredProduct[ 0 ].product_slug as string;
```

Comment:
Is there any chance that `filteredProduct` is an empty array? Maybe this would be safer as:

```js
return cart.products.reduce( ( slug, product ) => {
  if ( slug || ( product.uuid !== uuidToRemove ) ) {
    return slug;
  }
  return product.product_slug;
}, null );
```

With the above, the function would also be able to return `null`, so that case would have to be handled when it's used.

Reply:
With the refactor I mentioned in [this comment](https://github.com/Automattic/wp-calypso/pull/44236#discussion_r458840409), I could completely remove this function.